packages/perseus-editor/src/components/__tests__/start-coord-settings.test.tsx

```diff
@@ -42,26 +43,29 @@ describe("StartCoordSettings", () => {
         const heading = screen.getByText("Start coordinates");
 
         // Assert
-        expect(screen.getByText("Point 1")).toBeInTheDocument();
-        expect(screen.getByText("Point 2")).toBeInTheDocument();
+        expect(
+            screen.getByRole("button", {name: "Use default start coords"}),
+        ).toBeInTheDocument();
```

Comment:
I thought this would be more straightforward and less specific to one graph type, but I'm not sure how I feel about it. 🤔

Reply:
Oh!! 🤦🏽 I can check for the `aria-expanded` property. I think I did it this way to make sure that the correct stuff was showing up too, but I think checking `aria-expanded` makes more sense.